src/sidebar/components/FilterStatus.js

```diff
@@ -49,271 +26,235 @@ import { useRootThread } from './hooks/use-root-thread';
  */
 
 /**
- * @typedef FilterModeProps
- * @prop {FilterState} filterState
- * @prop {Thread} rootThread
- */
-
-/**
- * Render information about the currently-applied filters and a button that
- * allows the user to clear filters (or toggle user-focus mode on and off).
+ * Render status text describing the currently-applied filters.
  *
- * @param {FilterStatusPanelProps} props
+ * @param {FilterStatusMessageProps} props
  */
-function FilterStatusPanel({
-  actionButton,
+function FilterStatusMessage({
```

Comment:
This internal component used to be responsible for rendering a Card, filter description text and a button. Now it is solely tasked with the descriptive text and has been renamed accordingly. Subtractive. No changes to what remains.